utils/create-jobs.py

```diff
@@ -17,7 +17,7 @@ def parse_cmdline(machines, tests, rfs_types):
                         help="url fetch base",
                         default='https://download.automotivelinux.org/AGL/upload/ci/')
     parser.add_argument('--boot', action='store', dest='rfs_type', nargs = 1,
-                        choices=rfs_types, help='select boot type')
+                        choices=rfs_types, help='select boot type', default='ramdisk')
```

Comment:
As we discussed, boot should be required but no default provided. Right now only ramdisk works for qemu and nbd for the rest

Reply:
This was removed in a later commit after discussing with you